### PR TITLE
Use the host's /tmp, which should be big enough to extract the GOG in…

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
…staller there

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
bot, build

File: eu.vcmi.VCMI.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ finish-args:
   - --socket=pulseaudio
   - --share=network
   - --device=all # gamepad and dri
+  - --filesystem=/tmp # use the host's /tmp, which should be big enough to extract the GOG installer there
```
Comment (bbhtt) [inline eu.vcmi.VCMI.yaml:15]: Does setting TMPDIR as `XDG_RUNTIME_DIR/$FLATPAK_ID/tmp` or `/var/tmp` not work here?

`/var/tmp` would work as it is `~/.var/app/app-id/cache/tmp` but it is stored persistently then.

Comment (Alexander-Wilms) [reply eu.vcmi.VCMI.yaml:15]: The next patch will use `~/.local/share/vcmi/` and VCMI will delete any temporary files it creates there:

https://github.com/vcmi/vcmi/pull/4034

Comment (bbhtt) [reply eu.vcmi.VCMI.yaml:15]: Can you please import the patch? It's temporary until it becomes permanent :)

**Pull request merged**